docs/mxnet_backend/performance_guide.md

```diff
@@ -0,0 +1,106 @@
+
+```json
+"image_data_format": "channels_first", 
+"backend": "mxnet"
+```
+
+## Step 2. Update data to channels first format
+
+If your data is in the channels last format (Ex: shape=(256, 256, 3)), you need to convert the data to channels first format (Ex: shape=(3, 256, 256)). 
+
+Keras-MXNet has a utility API [`keras.utils.to_channels_first()`](https://github.com/awslabs/keras-apache-mxnet/blob/master/keras/utils/np_utils.py#L55) that can take a single input or a batch of data in channels last format and return back the data in channels first format.
+
+
+```
+NOTE:
+    Keras dataset API is not consistent across the different dataset. It returns few datasets based on the 
+    image_data_format in keras.json (Ex: CIFAR10 dataset] and few datasets always in the channels_last format (Ex: 
+    MNIST dataset). It is recommended to always verify the shape of data returned.
+```
+
+## Step 3. Update input_shape argument when constructing the model
+
+You need to update the input shapes in the network to channels first format. For example, using Conv2D on the color images with size 256x256, you need to set:
+
+For channels last format:
+
+```
+    model.add(Conv2D(32, (3, 3), padding='same', input_shape=(256,256,3)))
+```
+
+For channels first format:
+
+```
+    model.add(Conv2D(32, (3, 3), padding='same', input_shape=(3, 256,256)))
+```
+
+```
+NOTE:
+    It is recommended practice to pass the input_shape based on the shape field of your input tensor. For example:
+    
+    model.add(Conv2D(32, (3, 3), padding='same', input_shape=x_train.shape[1:]))
+
+```
+
+# Optimized MXNet binaries
+
+MXNet PyPi packages are distributed in multiple flavors each optimized for a specific underlying hardware. Please install the correct MXNet flavor optimized for your instance type for the best performance.
+
+1. Default: `pip install mxnet`
+ 
+2. Optimized for Intel CPUs with MKL: `pip install mxnet-mkl`
+
+3. Optimized for NVIDIA GPUs with CUDA9: `pip install mxnet-cu90` 
+
+4. Optimized for NVIDIA GPUs and Intel CPUs: `pip install mxnet-cu90-mkl`
+
+5. Refer installation guide for more configurations to build from source: [MXNet Installation guide](https://mxnet.incubator.apache.org/install/index.html) 
+
+Please see [MXNet Performance Guide](https://mxnet.incubator.apache.org/faq/perf.html) for more detailed performance tips for MXNet.
+
+
+# Using MXNet Profiler
+
+MXNet has a built-in profiler that provides detailed information about the execution times for each operator. MXNet profiler complements general profiling tools like `nvprof` and `gprof` by summarizing execution time at the operator level, rather than at a function, kernel, or instruction level. Follow [MXNet Performance Guide](https://mxnet.incubator.apache.org/faq/perf.html) for more details on MXNet profiler.
```

Comment:
...operator. The MXNet profiler...
Follow --> Review the MXNet...
delete "on MXNet profiler"